bfcfg

```diff
@@ -197,6 +198,219 @@ mfg_cfg()
   mfg_lock
 }
 
+add_trailing_null()
+{
+  for ((i = 0; i<${1}; i++))
+  do
+    printf '\0' >> ${2} 2>>${log_file}
+  done
+}
+
+mfg_ext_cfg()
+{
+  local tmp_file=/tmp/.bfcfg-mfg-ext-data
+  local MKCAP=/usr/lib/firmware/mellanox/boot/capsule/scripts/mlx-mkcap
+  local MFGEXT_CAPSULE=/tmp/MfgExtCapsule
+  local BFREC=/usr/bin/bfrec
+  local sys_mfr_sysfs=${efivars}/MfgExtSysMfr-${efi_global_var_guid}
```

Comment:
Let's have platform specific name for the EFI variables; I suggest the usage of the prefix "BfCfg" for example BfCfgSysMfr

Reply:
Done

> Would it be possible to reserve 256 bytes on top of the MFG EXT data for MFG data? If we don't do it now, it will cause compatibility issues in UEFI when we use capsules for MFG data.

I see, will do.